domain/model/service/modelservice.go

```diff
@@ -177,6 +206,111 @@ func (s *ModelService) GetModelCloudType(ctx context.Context) (string, error) {
 	return s.modelSt.GetModelCloudType(ctx)
 }
 
+// GetModelSummary returns a summary of the current model as a
+// [coremodel.ModelSummary] type.
+// The following error types can be expected:
+// - [modelerrors.NotFound] when the model does not exist.
+func (s *ModelService) GetModelSummary(
+	ctx context.Context,
+) (coremodel.ModelSummary, error) {
+	mSummary, err := s.controllerSt.GetModelSummary(ctx, s.modelUUID)
+	if err != nil {
+		return coremodel.ModelSummary{}, errors.Capture(err)
+	}
+
+	miSummary, err := s.modelSt.GetModelInfoSummary(ctx)
+	if err != nil {
+		return coremodel.ModelSummary{}, errors.Capture(err)
+	}
+
+	status := s.statusFromModelState(ctx, mSummary.State)
+	return coremodel.ModelSummary{
+		Name:           miSummary.Name,
+		UUID:           miSummary.UUID,
+		ModelType:      miSummary.ModelType,
+		CloudName:      miSummary.CloudName,
+		CloudType:      miSummary.CloudType,
+		CloudRegion:    miSummary.CloudRegion,
+		ControllerUUID: miSummary.ControllerUUID,
+		IsController:   miSummary.IsController,
+		OwnerName:      mSummary.OwnerName,
+		// TODO (tlm): Wire up when life is available.
+		Life:         life.Value(""),
```

Comment:
This will be available from the controller db, ie mSummary

Reply:
Ah I was waiting to see how we were going to wire this up to the model info service in general as it is needed from that perspective. There was talker of a worker dependency.

Reply:
Done & Done